keyboards/plywrks/lune/info.json

```
@@ -0,0 +1,75 @@
+{
+    "keyboard_name": "Lune",
+    "url": "",
+    "maintainer": "Ramon Imbao",
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                {"x":0.5, "y":0},
+                {"x":1.75, "y":0},
+                {"x":2.75, "y":0},
+                {"x":3.75, "y":0},
+                {"x":4.75, "y":0},
+                {"x":5.75, "y":0},
+                {"x":6.75, "y":0},
+                {"x":7.75, "y":0},
+                {"x":10, "y":0},
+                {"x":11, "y":0},
+                {"x":12, "y":0},
+                {"x":13, "y":0},
+                {"x":14, "y":0},
+                {"x":15, "y":0},
+                {"x":16, "y":0},
+                {"x":17, "y":0},
+
+                {"x":0.25, "y":1},
+                {"x":1.5, "y":1, "w":1.5},
+                {"x":3, "y":1}, {"x":4, "y":1},
+                {"x":5, "y":1}, {"x":6, "y":1},
+                {"x":7, "y":1}, {"x":9.5, "y":1},
+                {"x":10.5, "y":1}, {"x":11.5, "y":1},
+                {"x":12.5, "y":1}, {"x":13.5, "y":1},
+                {"x":14.5, "y":1}, {"x":15.5, "y":1},
+                {"x":16.5, "y":1, "w":1.5},
```

Strange formatting here.

```suggestion
                {"x":0.25, "y":1},
                {"x":1.5, "y":1, "w":1.5},
                {"x":3, "y":1},
                {"x":4, "y":1},
                {"x":5, "y":1},
                {"x":6, "y":1},
                {"x":7, "y":1},
                {"x":9.5, "y":1},
                {"x":10.5, "y":1},
                {"x":11.5, "y":1},
                {"x":12.5, "y":1},
                {"x":13.5, "y":1},
                {"x":14.5, "y":1},
                {"x":15.5, "y":1},
                {"x":16.5, "y":1, "w":1.5},
```